custom_components/bm6/translations/en.json

```diff
@@ -161,41 +161,41 @@
                 }
             },
             "state": {
-                "name": "Stan",
+                "name": "State",
                 "state": {
                     "unknown": "Unknown",
                     "ok": "OK",
                     "low_voltage": "Low voltage",
                     "under_voltage": "Too low voltage",
-                    "charging": "Landing",
+                    "charging": "Charging",
                     "idle": "Idle",
-                    "discharging": "Unloading",
+                    "discharging": "Discharging",
                     "over_voltage": "Too high voltage"
                 }
             },
             "rssi": {
-                "name": "Signal force (RSSI)"
+                "name": "Signal strength (RSSI)"
             },
             "device_percentage": {
-                "name": "Percentage of the device"
+                "name": "Device percent"
             },
             "device_state": {
-                "name": "Condition of the device",
+                "name": "Device state",
                 "state": {
                     "unknown": "Unknown",
                     "ok": "OK",
                     "low_voltage": "Low voltage",
-                    "charging": "Landing"
+                    "charging": "Charging"
                 }
             },
             "rapid_acceleration": {
```

Comment:
I'm not sure what these acceleration and deceleration values are. It looks like they haven't been implemented yet, but the units are set to mV/s in `sensor.py`, so I thought maybe they were actually charge/discharge rates?